docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Get started with security in Quarkus by using Basic authentication
 
 include::./attributes.adoc[]
 
-This guide demonstartes how you can quickly create a secure Quarkus application by using xref:security-built-in-authentication.adoc#basic-auth[Basic AuthenticationMechanism] and `JPA IdentityProvider` to create `SecurityIdentity` which is authorized by the `Role Based Access Control` (RBAC) layer before the access to application is permitted.
-
+To help you get started with security in Quarkus, this tutorial demonstrates how you can secure your Quarkus application endpoints by combining the xref:security-built-in-authentication.adoc#basic-auth[Quarkus built-in basic HTTP authentication] with the JPA identity provider.
+`IdentityProvider` verifies the authentication credentials to create a `SecurityIdentity` instance and authorizes secure access to the endpoint by using Role Based Access Control (RBAC). 
 [NOTE]
 ====
-Using `Basic AuthenticationMechanism` with `JPA IdentityProvider` is better than configuring the users and roles in `application.properties` therefore this guide recommends to combine `Basic AuthenticationMechanism` and `JPA IdentityProvider`. We will update this recommendation to use a reactive equivalent of the `JPA IdentityProvider` as soon as it is introduced.
+Completing these steps ensures that your Quarkus applications are more secure than just configuring users and roles in `application.properties` alone.
 ====
 
-This guide will conclude with recommending how to learn more about Quarkus Security, and in particular about its `OpenId Connect Authentication Mechanism`.
-
+After you have completed this tutorial, you will be ready to apply more advanced security mechanisms in Quarkus, for example, use the `OpenID Connect` authentication mechanism.
```

Comment:
Should we move prereqs section to immediately before the sequential steps begin? 

As a new Quarkus developer, I'm thinking I'd like to know what I will acheieve (be building) before I get into prereqs, unless the prereqs are extremely significanthere.